<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>1001</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font:19pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>450</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font:19pt &quot;Playbill&quot;;
</string>
      </property>
      <property name="text">
       <string>mot de passe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>540</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>confirmer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>540</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passe">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>460</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1001</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/imagge.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>confirmer</zorder>
     <zorder>pushButton</zorder>
     <zorder>id</zorder>
     <zorder>passe</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="employes">
      <property name="geometry">
       <rect>
        <x>-140</x>
        <y>-30</y>
        <width>1341</width>
        <height>741</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/imggg.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="EMPLOYES_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Employes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="COMMANDES_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Cmmandes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ALIMENTATION_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Alimentation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ANIMAUX_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Animaux</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EQUIPEMENTS_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>550</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
      </property>
      <property name="text">
       <string>Equipements</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="crud">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>781</width>
        <height>541</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>kj l </string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="liste">
       <attribute name="title">
        <string>Liste</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 24pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Employes</string>
        </property>
       </widget>
       <widget class="QLabel" name="back1_4">
        <property name="geometry">
         <rect>
          <x>-280</x>
          <y>-340</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="CHERCHER">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="chercher">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>330</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Chercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="CHER">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>320</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix2/img.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="aff3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>731</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QTableView" name="aff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>731</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="TRIER">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>290</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="verifier">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>290</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);
</string>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_pdf">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);
</string>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <zorder>back1_4</zorder>
       <zorder>label</zorder>
       <zorder>CHERCHER</zorder>
       <zorder>chercher</zorder>
       <zorder>CHER</zorder>
       <zorder>aff3</zorder>
       <zorder>aff</zorder>
       <zorder>TRIER</zorder>
       <zorder>verifier</zorder>
       <zorder>pb_pdf</zorder>
      </widget>
      <widget class="QWidget" name="ajouter">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="ajt_emp">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 23pt &quot;Playbill&quot;;

</string>
        </property>
        <property name="text">
         <string>Nouvel Employe</string>
        </property>
       </widget>
       <widget class="QLabel" name="ID">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="PRENOM1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>240</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="TEL1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>240</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="AGE">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="etat">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>300</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 21pt &quot;Playbill&quot;;

</string>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ETAT1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>300</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="email">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>300</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="prenom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AJOUTER1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>440</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="tel">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>230</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="EMAIL1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="age">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>170</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLabel" name="Nom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font:19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NOM1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLabel" name="back1_2">
        <property name="geometry">
         <rect>
          <x>-290</x>
          <y>-340</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="miseajour">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>440</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Mettre à jour</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SUPPRIMER1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NB_ANS1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>360</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="SAL1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>360</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="email_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Ans du travail</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>360</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="age_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>110</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Mot de passe </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="PASSE1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>120</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <zorder>back1_2</zorder>
       <zorder>ajt_emp</zorder>
       <zorder>TEL1</zorder>
       <zorder>AGE</zorder>
       <zorder>etat</zorder>
       <zorder>ETAT1</zorder>
       <zorder>AJOUTER1</zorder>
       <zorder>tel</zorder>
       <zorder>age</zorder>
       <zorder>ID</zorder>
       <zorder>ID1</zorder>
       <zorder>NOM1</zorder>
       <zorder>Nom</zorder>
       <zorder>prenom</zorder>
       <zorder>email</zorder>
       <zorder>EMAIL1</zorder>
       <zorder>PRENOM1</zorder>
       <zorder>miseajour</zorder>
       <zorder>SUPPRIMER1</zorder>
       <zorder>NB_ANS1</zorder>
       <zorder>SAL1</zorder>
       <zorder>email_2</zorder>
       <zorder>email_3</zorder>
       <zorder>age_2</zorder>
       <zorder>PASSE1</zorder>
      </widget>
      <widget class="QWidget" name="STATISTIQUE">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QLabel" name="back1">
        <property name="geometry">
         <rect>
          <x>-330</x>
          <y>-340</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="PLANNING">
       <attribute name="title">
        <string>Planning</string>
       </attribute>
       <widget class="QLabel" name="back1_3">
        <property name="geometry">
         <rect>
          <x>-720</x>
          <y>-570</y>
          <width>2271</width>
          <height>1601</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>381</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="ListWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>361</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="DISP">
       <attribute name="title">
        <string>Disponibilité</string>
       </attribute>
       <widget class="QLabel" name="back1_5">
        <property name="geometry">
         <rect>
          <x>-700</x>
          <y>-520</y>
          <width>2191</width>
          <height>1501</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="DIS">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Disponiblité</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="view">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>711</width>
          <height>441</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
